source/nodejs/adaptivecards-site/themes/adaptivecards/layout/designer.ejs

```diff
@@ -45,8 +45,27 @@
 	.acd-icon-scopeTemplate::before {
 		content: "\F2B0";
 	}
+
+	@media all and (display-mode: standalone) {
+		.header-holder {
+			display: none;
+		}
+	}
 </style>
 
+<script type="text/javascript">
+	// Register the service worker for PWA
+	if ('serviceWorker' in navigator) {
+		window.addEventListener('load', () => {
+			navigator.serviceWorker.register('/../service-worker.js', {scope: '/designer/'}).then(registration => {
```

Comment:
```suggestion
			navigator.serviceWorker.register('/service-worker.js', {scope: '/designer/'}).then(registration => {
```

You could either start with / (for the root) or ../ to move to the root from the subdirectory. I tend to prefer the former (site root relative links).